app/models/course/user_invitation.rb

```diff
@@ -39,4 +44,12 @@ def confirmed?
   def generate_invitation_key
     self.invitation_key ||= 'I'.freeze + SecureRandom.base64(8)
   end
+
+  # Sets the default for non-null fields.
+  # Currently sets the role attribute to :student is it's null.
+  #
+  # @return [void]
+  def set_defaults
+    self.role ||= :student
```

Comment:
Use `CourseUser.roles[:student]` instead - I think slightly clearer.

Reply:
There's an enum declaration within this class itself. I'll use that instead just to be consistent?

Reply:
Yup even better 👍